src/ophys_etl/workflows/on_prem/dags/ophys_processing.py

```diff
@@ -223,15 +226,33 @@ def run_inference():
         thumbnail_dir >> run_inference()
 
     @task_group
-    def trace_extraction(motion_corrected_ophys_movie_file):
-        run_workflow_step(
+    def trace_extraction(motion_corrected_ophys_movie_file,
+                         rois_file):
+        module_outputs = run_workflow_step(
             slurm_config_filename="trace_extraction.yml",
             module=TraceExtractionModule,
             workflow_step_name=WorkflowStepEnum.TRACE_EXTRACTION,
             workflow_name=WORKFLOW_NAME,
             docker_tag=app_config.pipeline_steps.trace_extraction.docker_tag,
             module_kwargs={
-                "motion_corrected_ophys_movie_file": motion_corrected_ophys_movie_file # noqa E501
+                "motion_corrected_ophys_movie_file": motion_corrected_ophys_movie_file, # noqa E501
+                "rois_file": rois_file
+            },
+        )
+
+        return module_outputs[WellKnownFileTypeEnum.ROI_TRACE.value]
+
+    @task_group
+    def demix_traces(motion_corrected_ophys_movie_file, roi_traces_file):
```

Comment:
I believe the original intent was to group all trace extraction steps into a single `task_group`, similar to how the `denoising` task group is structured. Currently each trace extraction step is separate. It doesn't make that much of a difference, but would allow to group all trace extraction steps in the UI into a single box.

Reply:
Right, I remember that now. I'll take a crack at putting them together into one. How does the name "trace_processing" for the task group sound?

Reply:
I like it